zerver/openapi/zulip.yaml

```diff
@@ -1287,6 +1309,7 @@ paths:
                     }
 
     patch:
+      operationId: update_user_by_id
```

Comment:
I think `update_user` is probably good enough?  Same with the next deactivate one.